kitsune/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -411,4 +411,70 @@ <h3 class="card--title">
       </div>
     </section>
   {% endif %}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro kb_subtopic_tabs(request, product, topic, subtopics) -%}
+  <nav class="tabs">
+    <ul class="tabs--list subtopics">
+      <li class="tabs--item">
+        <a class="tabs--link is-active" href="#kb-subtopic-all" data-event-category="link click">
+          <span>{{ _('View All') }}</span>
+        </a>
+      </li>
+      {% for subtopic in subtopics %}
+        <li class="tabs--item">
+          <a class="tabs--link" id="kb-subtopic-{{ subtopic.title }}" 
+             href="#kb-subtopic-{{ subtopic.title }}" 
+             data-event-category="link click" 
+             data-event-action="subtopic sidebar" data-event-label="{{subtopic.title}}">
+            <span>{{ pgettext('DB: products.Topic.title', subtopic.title) }}</span>
+          </a>
+        </li>
+      {% endfor %}
+    </ul>
+  </nav>
+
+  {% set all_docs, all_fallback_docs = kb_documents_for(request.LANGUAGE_CODE, topics=[topic], products=[product])%}
+  {#TODO: Add fallback docs rendering #}
+
+  <section class="topic-list" id="kb-tab-all">
+    {% for document in all_docs %}
+      <article class="topic-article">
+        <div class="topic-article--icon-wrap">
+          <img class="topic-article--icon" src="{{ webpack_static('protocol/img/icons/reader-mode.svg') }}" alt="" />
+        </div>
+        <div class="topic-article--text">
+          <h2 class="sumo-card-heading">
+            <a href="{{ document['url'] }}" data-event-category="link click" data-event-action="document"
+              data-event-label="{{ document['document_title'] }}">
```

Comment:
What do you think about using the following instead?
- `data-event-category="device-migration-tabbed-panel"`
- `data-event-action="link-click"`
- `data-event-label="{{document.title}}"`